package.json

```diff
@@ -16,7 +16,8 @@
     "PrettyCSS": "^0.3.10",
     "react": "^0.13.3",
     "reactify": "^1.1.1",
-    "watchify": "^3.2.3"
+    "watchify": "^3.2.3",
+    "i18next-client": "1.10.2"
```

Comment:
Nitpick, but I like to keep the dependencies alphabetized here : )

Also I prefer to use a `^`-prefixed version dependency rather than locking it into a specific version